public/styles/style.scss

```diff
@@ -0,0 +1,126 @@
+html {
+  font-size: 1em;
+  height: 100%;
+}
+
+body {
+  background-color: #ddd;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  height: 100vh
+}
+
+.main-button {
+  height: 35px;
+  align-self: center;
+  border-radius: 40px;
+  display: block;
+  border: 0;
+  width: 100%;
+  cursor: pointer;
+  background-color: #aaa;
+  box-shadow: 0 4px 2px #666;
+
+  &:hover {
+    background-color: #666;
+  }
+}
+
+.button-outer {
+  position: relative;
+  align-self: center;
+  text-align: center;
+  width: 50%;
+}
+
+.landing-page {
+  display: flex;
+  flex-direction: column;
+  align-content: space-between;
+  width: 100vw;
+  max-width: 600px;
+  height: 100%;
+}
+
+div {
+  border: 1px solid hotpink
+}
+
+.welcome-message {
+  background-color: #666;
+  height: 20%;
+  padding: 20px;
+  color: #ddd;
+
+  .welcome-hello { font-size: 1.5em; }
+  .welcome-question { font-size: 1.3em; }
+}
+
+.add-goal-button {
+  height: 10%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.container-list-goals {
+  height: 70%;
+  display: flex;
+  flex-direction: row;
+  background-color: #666;
+}
+
+.list-goals {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  margin: 0;
+  list-style: none;
+  width: 100%;
+  padding: 0;
+}
+
+.goal-item {
+  margin: 0;
+  display: flex;
+  flex-direction: row;
+  align-content: center;
+  width: 100%;
+}
+
+.goal-item-box {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  width: 100%;
+  border: 3px solid #ddd;
+  cursor: pointer;
+  height: 70px;
+
+  &:hover {
+    background-color: #aaa;
+  }
+
+  p {
+    font-size: 1.2em;
+    color: #ddd;
+
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+}
+
+.goal-item-img-container {
+  height: 70px;
+  width: 70px;
```

Comment:
Are these `70px` representing the same quantities? With sass you can define them in a single var